database/sql/src/main/scala/cromwell/database/slick/tables/MetadataEntryComponent.scala

```diff
@@ -26,15 +26,15 @@ trait MetadataEntryComponent {
 
     def metadataEntryId = column[Long]("METADATA_JOURNAL_ID", O.PrimaryKey, O.AutoInc)
 
-    def workflowExecutionUuid = column[String]("WORKFLOW_EXECUTION_UUID") // TODO: rename column via libquibase
+    def workflowExecutionUuid = column[String]("WORKFLOW_EXECUTION_UUID", O.Length(255)) // TODO: rename column via liquibase
 
-    def callFullyQualifiedName = column[Option[String]]("CALL_FQN") // TODO: rename column via libquibase
+    def callFullyQualifiedName = column[Option[String]]("CALL_FQN", O.Length(255)) // TODO: rename column via liquibase
```

Comment:
ToL: are these even real TODOs? I don't see this happening given the migration cost and what's wrong with the names anyway?

Reply:
preexisting and I agree with you, all this will go away in CaaS anyway

Reply:
It will?

Reply:
absolutely